plugins/cliconf/eos.py

```diff
@@ -193,7 +193,9 @@ def edit_config(
         commit=True,
         replace=None,
         comment=None,
+        **kwargs,
```

Comment:
my only concern around this change is the fact that the only keyword argument that exists is timer - and i think i would rather refactor all the calls to edit_config rather than introduce kwargs to this function.

Reply:
I agree it's a bit ugly, but if we change the interface and deviate from `CliconfBase`, then pylint fails. Not sure the object instance has any state that contains the timer details?

Reply:
I asked @Qalthos on Slack and they replied with a bit of insight:

https://ansiblenetwork.slack.com/archives/CEXQ0U2RH/p1689792762886269

Reply:
Looks like we might be able to do this in `edit_config`
```
timer = self.get_option("timer")
```
But not sure how to test this specific code path.